src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -0,0 +1,209 @@
+        $options = [];
+        $data = [];
+
+        if (isset($attributes['normalization_context']['groups'])) {
+            $options['serializer_groups'] = $attributes['normalization_context']['groups'];
+        }
+
+        if (isset($attributes['denormalization_context']['groups'])) {
+            $options['serializer_groups'] = isset($options['serializer_groups']) ? array_merge($options['serializer_groups'], $attributes['denormalization_context']['groups']) : $options['serializer_groups'];
+        }
+
+        foreach ($this->propertyNameCollectionFactory->create($resourceClass, $options) as $propertyName) {
+            $propertyMetadata = $this->propertyMetadataFactory->create($resourceClass, $propertyName);
+
+            if (
+                ($propertyMetadata->isReadable() && self::OUT_PREFIX === $io) ||
+                ($propertyMetadata->isWritable() && self::IN_PREFIX === $io)
+            ) {
+                $data[$name] = $this->parseProperty($resourceMetadata, $propertyMetadata, $io, null, $visited);
+            }
+        }
+
+        return $data;
+    }
+
+    /**
+     * Parses a property.
+     *
+     * @param ResourceMetadata  $resourceMetadata
+     * @param PropertyMetadata  $propertyMetadata
+     * @param string            $io
+     * @param Type|null         $type
+     * @param string[]          $visited
+     *
+     * @return array
+     */
+    private function parseProperty(ResourceMetadata $resourceMetadata, PropertyMetadata $propertyMetadata, $io, Type $type = null, array $visited = [])
+    {
+        $data = [
+            'dataType' => null,
+            'required' => $propertyMetadata->isRequired(),
+            'description' => $propertyMetadata->getDescription(),
+            'readonly' => !$propertyMetadata->isWritable(),
+        ];
+
+        if (null == $type) {
+            $type = $propertyMetadata->getType();
+
+            if (null === $type) {
+                // Default to string
+                $data['dataType'] = DataTypes::STRING;
+
+                return $data;
+            }
+        }
+
+        if ($type->isCollection()) {
+            $data['actualType'] = DataTypes::COLLECTION;
+
+            if ($collectionType = $type->getCollectionValueType()) {
+                $subProperty = $this->parseProperty($resourceMetadata, $propertyMetadata, $io, $collectionType, $visited);
+                if (self::IRI === $subProperty['dataType']) {
+                    $data['dataType'] = 'array of IRIs';
+                    $data['subType'] = DataTypes::STRING;
+
+                    return $data;
+                }
+
+                $data['subType'] = $subProperty['subType'];
+                $data['children'] = $subProperty['children'];
+            }
+
+            return $data;
+        }
+
+        $builtinType = $type->getBuiltinType();
+        if ('object' === $builtinType) {
+            $className = $type->getClassName();
+
+            if ('DateTime' === $className) {
```

Comment:
Can you replace this line with `if (is_subclass_of($class, \DateTimeInterface::class)) {` ?